app/jobs/application_job.rb

```diff
@@ -1,7 +1,7 @@
 class ApplicationJob < ActiveJob::Base
   # Automatically retry jobs that encountered a deadlock
   # retry_on ActiveRecord::Deadlocked
-
+  PASSWORD = "oh no its a password"
```

Comment:
🚫 **[Bearer]** <[ruby_lang_hardcoded_secret](https://docs.bearer.com/reference/rules/ruby_lang_hardcoded_secret)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>
# Hard-coded secret detected.
## Description

Applications should store secret values securely and not as literal values
in the source code.

## Remediations

✅ Retrieve secrets from a secure location at runtime

## Resources
- [OWASP hardcoded passwords](https://owasp.org/www-community/vulnerabilities/Use_of_hard-coded_password)
- [OWASP secrets management cheat sheet](https://cheatsheetseries.owasp.org/cheatsheets/Secrets_Management_Cheat_Sheet.html#21-high-availability)